apps/index.html

```diff
@@ -305,6 +305,22 @@
       </div>
 
       <div class="app">
+        <button onclick="install('118d77fa72eba6a685cb493bffb18fae?url=https://github.com/libreboard/libreboard/releases/download/v0.7.1/libreboard.spk')">Install »</button>
+        <h3>LibreBoard</h3>
+        <div class="app-details">
+          Originally by: <a href="https://github.com/yasaricli">Yaşar İçli</a> &amp; <a href="https://github.com/mquandalle">Maxime Quandalle</a><br>
```

Comment:
Nitpick: For other non-ports we say "Developed by:" instead of "Originally by:". :)

Otherwise I'm ready to merge this!